packages/flutter/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: parse_server_sdk_flutter
 description: Flutter plugin for Parse Server, (https://parseplatform.org), (https://back4app.com)
-version: 3.1.2
-homepage: https://github.com/phillwiggins/flutter_parse_sdk
+version: 3.2.0
```

Comment:
Incorrect version; if this PR only contains bug fixes as indicated in the changelog, then the next flutter version will be `3.1.3`.